requirements.txt

```diff
@@ -1,3 +1,7 @@
 pyjwt==1.0.0
 blinker==1.3
 cryptography>=0.8.1
+contextlib2==0.5.4
+testscenarios==0.5.0
+mock==2.0.0
```

Comment:
IMHO, requirements that are only needed for tests should go in a separate `test_requirements.txt` file, not part of the main `requirements.txt`